### PR TITLE
Various Text Changes

### PR DESCRIPTION
**Pull request by PotatoTomahto**
## About The Pull Request
Text:
- Records work damage written in all caps
- Queue console outputs have different styles to make them more noticeable
- Quirk descriptions made more consistent
- "Show Attributes" changed to "View Attributes" so its sorted next to "View Gifts"

Makes all telecomms and some LCorp machines indestructible.

## Why It's Good For The Game
Lots of minor text or description changes I wanted. Also stop breaking tcomms.

## Changelog
:cl:
tweak: lots of text, roulette
/:cl:

### CODE REVIEW

**Review by LanceSmites328 — DISMISSED** [review]

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Stop fucking with resistance to text. We talked about this before, and Resistant is a more clear name than ineffective.

I really don't like how this removes any sort of obscurity with the armor values, giving people raw values is an invitation to use math. I don't want to encourage people calculating exactly how much damage they'd do with X weapon vs Y weapon, as it feels EXTREMELY gamey.

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Atomize PR.

File: code/__HELPERS/abnormalities.dm [hunk 1]
```diff
@@ -2,21 +2,21 @@
 /proc/SimpleResistanceToText(resist = 1)
 	switch(resist)
 		if(0 to 0) //Just putting 0 doesn't work.
```
Comment (Kitsunemitsu) [inline code/__HELPERS/abnormalities.dm:4]: This is not how switches work. It was fine before.

File: code/game/machinery/computer/abnormality_queue.dm [hunk 2]
```diff
@@ -90,15 +90,15 @@
...
+	to_chat(usr, "<span class='boldnotice'>[initial(target_type.name)] has been selected.</span>")
 	playsound(get_turf(src), 'sound/machines/terminal_prompt_confirm.ogg', 50, TRUE)
 	log_game("[usr] has [logstring] the anomaly to [initial(target_type.name)].")
 	if(lock_after)
 		message_admins("[usr] has [logstring] the anomaly to [initial(target_type.name)].")
 		locked = TRUE
 		// PE awarded for yellow roll - just as kirie had wanted.
 		SSlobotomy_corp.available_box += 250
-		to_chat(usr, "<span class='notice'>The headquarters reimbursed you the costs of extracting a specific abnormality. You will receive a random one.</span>")
+		to_chat(usr, "<span class='boldnotice'>A random Abnormality has been selected. LobCorp HQ has reimbursed you for the costs of extracting a specific Abnormality.</span>")
```
Comment (Kitsunemitsu) [inline code/game/machinery/computer/abnormality_queue.dm:101]: Don't use LobCorp, in lore it's L Corp

File: code/game/machinery/roulette_machine.dm [hunk 3]
```diff
@@ -54,6 +54,11 @@
 	jackpot_loop = new(list(src), FALSE)
 	wires = new /datum/wires/roulette(src)
 
+/obj/machinery/roulette/Destroy()
```
Comment (Kitsunemitsu) [inline code/game/machinery/roulette_machine.dm:57]: Why is any of this in this PR?

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Don't put text changes and a fix PR into one.
If you REALLY want to put up the resistance to text PR, make it it's own PR, I however, will likely not merge a needless change such as this, to overhaul a system to use more vague wording, and encouraging the players to use math.

The Text changes and roulette should honestly two separate PRs.

**Review by LanceSmites328 — APPROVED** [review]

**Review by Kitsunemitsu — APPROVED** [review]
Looks good to me.

**Pull request merged**